.circleci/config.yml

```diff
@@ -370,15 +365,11 @@ workflows:
           filters: *filter_rc_tags
           requires:
             - rc_e2e_tests
-      - anchore/image_scan:
+      - grype/list-image-vulns:
           context: dockerhub
-          filters: *filter_rc_tags
           requires:
             - push_rc_image
-          image_name: anchore/anchore-cli-dev:${CIRCLE_TAG}
-          private_registry: true
-          timeout: '2000'
-          policy_failure: true
+          image-name: anchore/anchore-cli-dev:rc
```

Comment:
This can use ${CIRCLE_TAG} for the image tag.

Reply:
updated. thanks!